.gitignore

```diff
@@ -7,19 +7,14 @@ target
 
 # BlueJ files
 *.ctxt
-.gradle
 .idea
 ParkingServer.iml
-gradlew.bat
 build
-gradlew
-gradle
 # Mobile Tools for Java (J2ME)
 .mtj.tmp/
 
 
 # Package Files #
-*.jar
```

Comment:
You should leave this in your .gitignore. 

gradle wrapper jar is the only file we are interested to commit. So, just rollback this row

Reply:
✅